examples/gltf/package.json

```diff
@@ -9,12 +9,12 @@
     "open-vr": "adb reverse tcp:8080 tcp:8080 && adb shell am start -a android.intent.action.VIEW -d http://localhost:8080/"
   },
   "dependencies": {
-    "@loaders.gl/core": "^1.2.0",
-    "@loaders.gl/draco": "^1.2.0",
-    "@luma.gl/addons": "^7.2.0",
-    "@luma.gl/constants": "^7.2.0",
-    "@luma.gl/core": "^7.2.0",
-    "math.gl": "^3.0.0-beta.2"
+    "@loaders.gl/core": "^1.0.0",
+    "@loaders.gl/draco": "^1.0.0",
+    "@luma.gl/addons": "^7.1.0-beta.1",
+    "@luma.gl/constants": "^7.1.0-beta.1",
+    "@luma.gl/core": "^7.1.0-beta.1",
+    "math.gl": "^3.0.0-alpha.6"
```

Comment:
why downgrade dependencies?

Reply:
Good catch! this probably happened because I split the original PR into three PRs and forgot to clean these out.